### DISCUSSION TITLE
Welcome to Makelangelo-software Discussions!

### DISCUSSION THREAD

**Discussion by i-make-robots** [MarginallyClever/Makelangelo-software#371] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PPAC37** [comment]
Greeting !

With a background in programming, I fell into 3D printing for my personal enjoyment for several years.
I started with a K8400 printer which was quite difficult to master, I learned on the job and I am starting to roughly know how to configure and compile a Marlin 2.0.x firmware.

Having accumulated 3d printer parts, building a polargraph was one way to use them ... my trial was suspended due to firmware barely matching (not without firmware changes, out of my reach) the hardware I had .

And from there ... after like one year on a French forum on 3D printers I discovered that version 2.0.9.2 of marlin can be configured for polargraph and will solve the difficulties I had encountered (printing from the card SD in stand-alone mode with a RepRap 128x64 LCD screen.)

So here I am :)
Marlin-polargraph + Makelangelo-Software are great projects.

Sorry in advance, I am French and I cannot make a sentence written without mistakes in my native language so it is complicated in English.

Here I hope to learn and share knowledge. And help if I can.

**Comment by i-make-robots** [comment]
Ton englais est excellent.  Mon francais est tres rouille.

**Comment by PPAC37** [comment]
Hope to improve myself, because at the moment I am using google translate and reiterating the translation, changing the translated language, to both get something that seems to make sense in both languages, and that stays more or less constant.

**Comment by i-make-robots** [comment]
![Capture5](https://user-images.githubusercontent.com/1464454/146994933-e0edc095-053e-4dd9-ab64-b02ec67af35f.PNG)

**Comment by PPAC37** [comment]
A documented bug becomes a feature;)
